### DISCUSSION TITLE
AWS Region support

### DISCUSSION THREAD

**Discussion by Chandramouli15** [cloudforet-io/community#147] [General]
Cloudforet support aws UAE region (me-central-1) ?

**Comment by ImMin5** [comment]
Hi @Chandramouli15 

Thank you for opening this discussion.

Are you referring to the AWS Inventory Collector?
Yes, we currently support the UAE region (me-central-1).
Please check the link below for more information.


**https://github.com/cloudforet-io/plugin-aws-inven-collector/blob/c75523bcb510fa662aca58b79c3ed242a2ca5b53/src/plugin/conf/cloud_service_conf.py#L317**
